app/src/main/java/ingsw/group1/findmyphone/cryptography/RandomStringGenerator.java

```diff
@@ -0,0 +1,53 @@
+package ingsw.group1.findmyphone.cryptography;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+/**
+ * Class used to generate random strings and int value within the bounds.
+ *
+ * @author Pardeep Kumar
+ */
+public class RandomStringGenerator {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'RandomStringGenerator' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4958670)